spec/requests/api/sessions/destroy_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'DELETE api/v1/users/sign_out', type: :request do
+  let(:user) { create(:user) }
+
+  subject(:sign_out) { delete destroy_user_session_path, headers: auth_headers, as: :json }
+
+  context 'destroying a valid user session' do
+    it 'returns success' do
+      sign_out
+      expect(response).to be_successful
+    end
+
+    it 'returns invalid token authentication data' do
+      sign_out
+      token = response.header['access-token']
+      client = response.header['client']
+      expect(token).to be_nil
+      expect(client).to be_nil
+    end
+
+    it 'removes user token' do
+      headers = auth_headers
+      expect {
+        delete destroy_user_session_path, headers: headers, as: :json
```

Comment:
why not `sign_out`?

Reply:
For some reason, when using the subject or a let, the test fails because it doesn't detect the expected change. That's why I had to leave the code like this (this way it does detect the change and it doesn't fail).